zipkin-ui/static/traces_it_IT.properties

```diff
@@ -0,0 +1,29 @@
+traces.servicename = Nome del servizio
+traces.spanname = Nome dello Span
+traces.stime = Inizio
+traces.etime = Fine
+traces.duration = Durata (μs) >=
+traces.limit = Limite
+traces.findBtn = Trova tracce
+traces.sort = Ordina
+service-percentage-desc = Service Percentage: Longest First
+service-percentage-asc = Service Percentage: Shortest First
+duration-desc = Piú lunghi prima
+duration-asc = Piú corti prima
+timestamp-desc = Nuovi prima
+timestamp-asc = Vecchi prima
+traces.des = Seleziona i criteri per ricercare le tracce.
+modal.title = Come trovare una traccia
+modal.p1 = Stai cercando una traccia eh? Sei nei posto giusto!
+modal.title2 = Nome servizi
+modal.p2 = Prima di tutto seleziona un sevizio per cui cercare. Tutte le tracce sono indicizzate per nome del servizio.
+modal.p3 = Once that is done you have three options to pick from. See notes on each below.
+modal.p3 = Quando hai fatto hai tre opzioni tra cui scegliere. Guarda le note per ognuna di loro sotto.
+modal.title3 = 1. Ricerca per nome dello span
+modal.p4 = Span sono generalmente metodi thrift o endpoints Rails. Questo ti permette di recuperare tracce che accedono ad una particolare parte del servizio.
+modal.title4 = 2. Ricerca per annotazione
+modal.p5 = Un annotazione é un punto nel tempo all'iterno della traccia. é un valore di tipo stringa, questo e quello che puoi utilizzare come parametro di ricerca. Possono essere come tipo
```

Comment:
- Un'annotazione
- iterno --> interno

I'd say: "Il suo valore e' di tipo stringa ed e' quello che puoi utilizzare come parametro di ricerca. Possono essere cose come"

Reply:
Btw, I know that the English version is the same, but is it normal that the sentence doesn't end?

Reply:
"<strong>finagle.timeout</strong>" follows this, so yeah abnormal.. maybe comment deserving? or perhaps better to scrap the strong and inline the annotation it refers to